packages/core/stories/patterns/contact-details/contact-details.stories.tsx

```diff
@@ -0,0 +1,193 @@
+  avatarSrc = "https://storybook.saltdesignsystem.com/assets/avatar-840503bf.png",
+}: ContactDetailsProps) => {
+  return (
+    <StackLayout direction={direction} gap={gap}>
+      <Avatar
+        src={avatarSrc}
+        aria-label="Woman Avatar"
+        fallbackIcon={<WomanIcon />}
+        size={avatarSize}
+      />
+      <StackLayout direction={"column"} gap={gap}>
+        <Text styleAs="h3"> {basicContact.primary} </Text>
+        <Text styleAs="h4"> {basicContact.secondary}</Text>
+        <StackLayout gap={gap} direction={direction}>
+          {basicContact.metadata.map((metadata) => {
+            return (
+              <Text styleAs="label" key={metadata.label}>
+                {metadata.icon} {metadata.label} : {metadata.value}
+              </Text>
+            );
+          })}
+        </StackLayout>
+      </StackLayout>
+    </StackLayout>
+  );
+};
+
+export const Horizontal = () => {
+  return <ContactDetails avatarSize={2} gap={1} direction="row" />;
+};
+
+export const WithIcon = () => {
+  return <ContactDetails avatarSrc="" />;
+};
+
+export const CardEmbedded = () => {
+  return (
+    <Card>
+      <ContactDetails />
+    </Card>
+  );
+};
+
+export const QuickAction = () => {
+  return (
+    <StackLayout direction="row" gap={1}>
+      <Avatar
+        src="https://storybook.saltdesignsystem.com/assets/avatar-840503bf.png"
+        aria-label="Woman Avatar"
+        fallbackIcon={<WomanIcon />}
+        size={2}
+      />
+      <StackLayout direction={"column"} gap={1}>
+        <Text styleAs="h3"> {basicContact.primary} </Text>
+        <Text styleAs="h4"> {basicContact.secondary}</Text>
+        <StackLayout gap={1} direction={"row"} separators>
+          <Tooltip content={"send email"}>
+            <Button variant="secondary">
+              <MessageIcon />
+            </Button>
+          </Tooltip>
+          <Tooltip content={"make call"}>
+            <Button variant="secondary">
+              <CallIcon />
+            </Button>
+          </Tooltip>
+          <Tooltip content={"send text"}>
+            <Button variant="secondary">
+              <ChatIcon />
+            </Button>
+          </Tooltip>
+        </StackLayout>
+      </StackLayout>
+    </StackLayout>
+  );
+};
+
+export const CollapsableDetails = () => {
+  const [toggleData, setToggleData] = useState(false);
+
+  function handleClick() {
+    setToggleData(!toggleData);
+  }
+  return (
+    <StackLayout direction="row" gap={1}>
+      <Avatar
+        src="https://storybook.saltdesignsystem.com/assets/avatar-840503bf.png"
+        aria-label="Woman Avatar"
+        fallbackIcon={<WomanIcon />}
+        size={2}
+      />
+      <StackLayout direction={"column"} gap={1}>
+        <Text styleAs="h3"> {basicContact.primary} </Text>
+        <Text styleAs="h4"> {basicContact.secondary}</Text>
+        <StackLayout gap={1} direction={"row"}>
+          <Tooltip content={"send email"}>
+            <Button variant="secondary">
```

Comment:
Just to add a note, all of the buttons need to be updated following the icon button guidance

Reply:
Buttons have a tooltip, and i've updated the tooltip text - was there anything additional ?

Reply:
"When a button has an icon only and no text, pass an aria-label to the button that describes its purpose." Icons also need aria-hidden applied to them when this is done